templates/deployment.yaml

```diff
@@ -24,6 +24,11 @@ spec:
             - name: TALISKER_NETWORKS
               value: 10.0.0.0/8
 
+            - name: SECRET_KEY
+              secretKeyRef:
+                key: {{ data.get('name') or get_site_name() }}
+                name: secret-keys
```

Comment:
If you see above some deployments are using `secret-key`, please just make sure all is now set under `secret-keys`

Reply:
yeah I removed all the secret keys from the other services so it uses now the new way of storing the secret keys.

My next task will be to remove manually all the secret keys already in the k8s cluster.